### PR TITLE
Created new buttonAs example, removed unused scss file and removed old experiments code

### PR DESCRIPTION
**Pull request by micahgodbolt**
Fixed buttonAs interface to work with buttons. Added example using buttonAs, and removing old unused files. 

#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/CommandBar/examples/CommandBar.ButtonAs.Example.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  constructor(props: ICommandBarProps) {
+    super(props);
+  }
+
+  public render(): JSX.Element {
+
+    const customButton = (props: IButtonProps) =>
+      <CommandBarButton
+        onMouseEnter={ () => console.log(`${props.text} hovered`) }
```
Comment (cliffkoh) [inline packages/office-ui-fabric-react/src/components/CommandBar/examples/CommandBar.ButtonAs.Example.tsx]: Even though this is but just an example, we should probably still adhere to the documented best practices and avoid inline functions.

File: packages/office-ui-fabric-react/src/components/CommandBar/examples/CommandBar.ButtonAs.Example.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+
+import { CommandBar, ICommandBarProps } from 'office-ui-fabric-react/lib/CommandBar';
+import { CommandBarButton, IButtonProps } from 'office-ui-fabric-react/lib/Button';
+
+
```
Comment (MLoughry) [inline packages/office-ui-fabric-react/src/components/CommandBar/examples/CommandBar.ButtonAs.Example.tsx]: Build is failing due to multiple consecutive blank lines

**Comment by micahgodbolt** [comment]
Fixed.


Sent from Outlook<http://aka.ms/weboutlook>

________________________________

**Comment by micahgodbolt** [comment]
@cliffkoh yeah, caught that in the last lint run. All cleaned up and passing.

**Pull request merged**